html/webappapis/structured-clone/structured-clone.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<title>structuredClone() tests</title>
+<meta charset=utf-8>
+<div id=log></div>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script src=/html/webappapis/structured-clone/structured-clone-battery-of-tests.js></script>
+<script src=/html/webappapis/structured-clone/structured-clone-battery-of-tests-with-transferables.js></script>
+<script src=/html/webappapis/structured-clone/structured-clone-battery-of-tests-harness.js></script>
+<script>
+  // REMOVE ME
+  function structuredClone(obj, transferList) {
+    return JSON.parse(JSON.stringify(obj));
+  }
+  structuredCloneBatteryOfTests({
+    structuredClone: structuredClone
+  });
```

Comment:
This needs to be adjusted I think to account for the fact that the second argument needs to be destructured, right?

Reply:
Looking at [this](https://github.com/web-platform-tests/wpt/blob/e1ea3e0096713632e94cf2141bd72ab9db4a20b7/html/webappapis/structured-clone/structured-clone-battery-of-tests-harness.js#L1-L22), the invocation seems correct to me. But it seems the function name was changed since I opened this :D

Reply:
That wants a transferList whereas the specification method is passed a dictionary that contains a member whose value is a transferList, right?

Reply:
`structuredCloneBatteryOfTests` expects a dictionary. The dictionary has a bunch of optional keys, and `structuredClone` as the only _mandatory_ key. The `structuredClone` key is expected to map to a function that takes two arguments: The object to be structured cloned and an optional transfer list and returns the structured clone.

Reply:
Right, and what's the signature of the standardized `structuredClone()` method?

Reply:
Ooooh, now I am with you. Didn’t realize the function got changed in whatwg/html#3414. Apologies. I’ll fix that!